packages/server/modules/core/utils/dbNotificationListener.ts

```diff
@@ -123,12 +122,24 @@ const reconnect = async () => {
 
     dbNotificationLogger.info('Attempting to (re-)connect...')
 
-    const newConnection = await (knex.client as Knex.Knex.Client).acquireRawConnection()
+    // creating externally managed PG connection from knex mainDB connection settings
+    const newConnection = new Client(
+      (mainDb.client as Knex.Knex.Client).connectionSettings
```

Comment:
@iainsproat this contained keys like (ssl, application_name, connectionString) locally, which I assume is also gonna be the case in prod